src/Illuminate/Cookie/CookieJar.php

```diff
@@ -30,6 +30,13 @@ class CookieJar implements JarContract
     protected $secure = false;
 
     /**
+     * The default httpOnly setting (defaults to true).
+     *
+     * @var bool
+     */
+    protected $httpOnly = true;
```

Comment:
The default value was true, and it can be override by passing another value to make function. Now all we make that we read  this value from session config